libs/components/indicators/src/lib/modules/wait/wait.component.ts

```diff
@@ -36,6 +36,25 @@ export class SkyWaitComponent implements OnInit {
       this.adapterService.removeWaitBounds(this.elRef);
     }
 
+    const blockFocus = this._isFullPage || !this.isNonBlocking;
+    if (blockFocus) {
+      if (value) {
+        // Before wait, capture the active element.
+        if (
+          (this.elRef.nativeElement.parentElement as HTMLElement).matches(
```

Comment:
Any DOM calls should probably be in the adapter

Reply:
Moved the changes to the adapter.